modules/exploits/multi/http/wp_royal_elementor_addons_rce.rb

```diff
@@ -0,0 +1,140 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HTTP::Wordpress
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'WordPress Royal Elementor Addons RCE',
+        'Description' => %q{
+          Exploit for the unauthenticated file upload vulnerability in WordPress Royal Elementor Addons and Templates plugin (< 1.3.79).
+        },
+        'Author' => [
+          'Fioravante Souza', # Vulnerability discovery
+          'Valentin Lobstein' # Metasploit module
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2023-5360'],
+          ['WPVDB', '281518ff-7816-4007-b712-63aed7828b34']
+        ],
+        'Platform' => ['unix', 'linux', 'win', 'php'],
+        'Arch' => [ARCH_PHP, ARCH_CMD],
+        'Targets' => [['Automatic', {}]],
+        'DisclosureDate' => '2023-11-23',
+        'DefaultTarget' => 0,
+        'Privileged' => false,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [IOC_IN_LOGS]
+        }
+      )
+  )
+
+    register_options([
+      OptString.new('TARGETURI', [true, 'The base path to the WordPress application', '/']),
+      OptBool.new('SSL', [true, 'Use SSL/TLS connection', true]),
```

Comment:
I believe these option will already be included. You may need to set `SSL` to true in default options.
```suggestion
```